tests/pages/integration-template.html

```diff
@@ -118,29 +118,29 @@
   window.snowplow('trackTrans');
 
   var testAcceptRuleSet = {
-    accept: ['iglu:com.acme_company/*/jsonschema/*-*-*']
+    accept: ['iglu:com.snowplowanalytics.snowplow/ad_impression/jsonschema/*-*-*']
```

Comment:
If we can, it would be good to preserve the rule having a wildcard mid-path for at least one of the rulesets. 

Can imagine a scenario where someone makes an innocuous-looking change which messes with on that front. This is one that would have a big impact if not caught till production since they might be attaching global contexts to a lot of different events.

Let me know if there's some challenge to doing this that I don't see - if we need to we can leave this as is and just log an issue to solve for this when we improve the tests themselves.

Reply:
Good plan. I think the mid path just got lost in my search and replace frenzy.

I've put them both back and everything seems to still be ok.